docs/USAGE-GUIDE.md

```diff
@@ -1,71 +1,22 @@
 # Using s2n-tls
 
-s2n-tls is a C library, and is built using Make. To clone the latest
+s2n-tls is a C library, and is built using CMake. To clone the latest
 copy of s2n-tls from git use:
 
-```shell
+```bash
 git clone https://github.com/aws/s2n-tls.git
 cd s2n-tls
 ```
```

Comment:
These are pretty basic instructions to begin with, and now we've written them out in two different files. They should probably either live here or in BUILD.md, but not both. Or maybe nowhere, if we assume someone using github knows how to clone from github :)

Reply:
lol it was actually three files since I put a clone in the README as well. I removed this from BUILD and USAGE-GUIDE, but kept the one in the README so the whole quickstart is copy-paste. If you prefer it's removed from the README as well let me know.

Reply:
I missed the one in the README! But I think you're right, and it can stay there.